lib/easypost/version_info.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
```

Comment:
suggestion: Why not dump this into our already existing `util.rb` file since it's a utility?

Reply:
Sure